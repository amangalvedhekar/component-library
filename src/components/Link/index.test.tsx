import React from 'react';
import { shallow, mount } from 'enzyme';
import {Link} from "./";

describe("Link Component", () => {
    it("should render", () => {
        const wrapper = shallow(<Link fontWeight={"bold"}>Some Link</Link>);
        expect(wrapper).toMatchSnapshot();
    });
    it("should have href set to default if not provided", () => {
        const wrapper = mount(<Link fontWeight={"bold"}>Another Link</Link>);
        const elem = wrapper.find('a');
        expect(elem.props()).toHaveProperty("href", "#");
    });
});