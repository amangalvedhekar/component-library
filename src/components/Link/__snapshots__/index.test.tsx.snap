// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Link Component should render 1`] = `
<styled.a
  fontWeight="bold"
  href="#"
  target="_self"
>
  Some Link
</styled.a>
`;
